### ISSUE TITLE
Change the cluster style

### ISSUE THREAD

**Issue by BertCanavaggio** [infusion-code/angular-maps#73]
Hi there,

Can we change the cluster icon and add a svg instead of the default image:

<img width="83" alt="screen shot 2018-08-09 at 20 36 38" src="https://user-images.githubusercontent.com/17899996/43918685-00f8dc26-9c14-11e8-93e6-fded11e947bc.png">

So far I tried to play with the inputs IconInfo, ClusterIconInfo and Styles without success.

Thanks!

**Comment by thor-schueler** [comment]
Yes, that is certainly possible. IconInfo will change the image for an individual marker, ClusterIconInfo for a cluster. Take a look at the sample page, it has samples for custom marker images. Take a look at https://github.com/infusion-code/angular-maps/blob/master/src/components/cluster-layer.ts also, it show how to dynamically create an SVG cluster icon based on the data (for the dynamic stock marker images) (CreateDynamicSizeMarker)

To use custom cluster SVG, you have to implement a callback and set it in x-cluster-layer CustomMarkerCallback. The callback is (m: Array<Marker>, i: IMarkerIconInfo) => string. This will be called for each cluster and the arguments are an array with  the markers in the cluster and the MarkerIconInfo that you have set using the IconInfo property. You can use this method to generate arbitrary svg based on the data passed in (I am usually using this to to pie chart clusters or similar). The return value of the callback should be either an SVG string or a data: string for an image, either way. 

If you do not need icons that change depending on the content of the cluster, you can just provide the SVG string in the url property of the IconInfo, specify MarkerTypeId.RotatedImageMarker and use IconInfo.rotation = 0. I've not tried this, but it should work. The logic for creating a marker can be found in https://github.com/infusion-code/angular-maps/blob/master/src/models/marker.ts 

Hope that helps. Let me know...
